presto-docs/README.md

```diff
@@ -0,0 +1,73 @@
+# presto-docs - Presto Documentation
+
+The presto-docs module contains the reference documentation for Presto.
+
+## Tools
+
+The engine used to create the documentation in HTML format is the Python-based
+[Sphinx](https://www.sphinx-doc.org). Installation instructions for various OS
+are [available on the Sphinx site](https://www.sphinx-doc.org/en/master/usage/installation.html).
+
+Documentation source files can be found in [Restructured
+Text](https://en.wikipedia.org/wiki/ReStructuredText) (`.rst`) format in
+`src/main/sphinx` and sub-folders.
+
+In addition, `make` is required
+
+## Build and Read
+
+With the tools installed and available on the PATH, you can build the docs
+easily with make:
+
+```bash
+cd presto/presto-docs
```

Comment:
(i like to provide instructions relative to project root)

`presto-docs` is guaranteed to exist, but the person reading this may or may not have `presto` directory, depending how they checked out the sources

Reply:
Fixed